### PR TITLE
Fix some typos in documentation

### PR DESCRIPTION
**Pull request by Miciah**

### CODE REVIEW

File: documentation/oo_deployment_guide_comprehensive.adoc [hunk 1]
```diff
@@ -2161,9 +2161,9 @@ service cgred restart
 In order for cgroups to work correctly, you need to ensure that services are started in the correct order.
 
 === Configure Disk Quotas
-When a consumer of OpenShift Origin creates an application gear, you will need to be able to control and set the amount of disk space that the gear can consume. This configuration is located in the _/etc/openshift/resource_limits.conf_ file. The two settings of interest are the qouta_files and the quota_blocks. The usrquota setting specifies the total number of files that a gear / user is allowed to own. The quota_blocks is the actual amount of disk storage that the gear is allowed to consume — where 1 block is equal to 1024 bytes.
```
Comment (Miciah) [inline documentation/oo_deployment_guide_comprehensive.adoc]: The line is too long for GitHub to highlight the change.  The change was "qouta_files" to "quota_files".

File: documentation/oo_deployment_guide_puppet.adoc [hunk 2]
```diff
@@ -975,7 +975,7 @@ Default: if ose_version == undef '/assets/logo-origin.svg'
 OpenShift Instance Name
 
 Default: if ose_version == undef 'OpenShift Origin'
-         if ose_version != undef 'Openshift Enterprise'
```
Comment (Miciah) [inline documentation/oo_deployment_guide_puppet.adoc]: I did verify that we spell it correctly in the Puppet module.

File: documentation/oo_install_users_guide.adoc [hunk 3]
```diff
@@ -456,7 +456,7 @@ At this point, all that remains to configure your OpenShift deployment is to tel
 The installer supports three different ways to set subscription preferences. This is great from a deployment flexibility perspective, but may be really confusing if this is your first time through an OpenShift deployment.
 
 ==== Pathway #1 - Command Line Options
-The first way to set subscription preferences is via the command line. This option works well if you are building an unattended installation system and you want to dyanamically set this information. The settings provided at the command line will trump any conflicting settings from the installer config file. But if you are _not_ running an unattended installation, you'll still have the opportunity to override these values. Bottom line: don't set subscription information on the command line unless you are using it in conjunction with an unattended installation (see the link:#w-workflow-workflow_id[-w] command line argument for more information).
```
Comment (Miciah) [inline documentation/oo_install_users_guide.adoc]: The change was "dyanamically" to "dynamically".

**Comment by Miciah** [comment]
Looks like some of the files I modified have been deleted.  Rebasing.

File: documentation/oo_cartridge_guide.adoc [hunk 4]
```diff
@@ -1076,7 +1076,7 @@ logs/          Log data (like httpd access/error logs)
 lib/           Various libraries
 bin/setup      The script to setup the cartridge
 bin/build      Default build script
-bin/teardown   Called at cartridge descruction
+bin/teardown   Called at cartridge destruction
 bin/control    Init script to start/stop httpd
 versions/      Version data to support multiple python versions (copied into place by setup
```
Comment (tiwillia) [inline documentation/oo_cartridge_guide.adoc]: While we are at it, this is missing a `)` at the end.

Comment (Miciah) [inline documentation/oo_cartridge_guide.adoc]: Ah, right! Thanks!

File: documentation/oo_deployment_guide_comprehensive.adoc [hunk 5]
```diff
@@ -2161,9 +2161,9 @@ service cgred restart
...
+When a consumer of OpenShift Origin creates an application gear, you will need to be able to control and set the amount of disk space that the gear can consume. This configuration is located in the _/etc/openshift/resource_limits.conf_ file. The two settings of interest are the quota_files and the quota_blocks. The usrquota setting specifies the total number of files that a gear / user is allowed to own. The quota_blocks is the actual amount of disk storage that the gear is allowed to consume — where 1 block is equal to 1024 bytes.
```
Comment (tiwillia) [inline documentation/oo_deployment_guide_comprehensive.adoc]: I think you meant `The quota_files setting sepcifies...`?

Comment (Miciah) [inline documentation/oo_deployment_guide_comprehensive.adoc]: Thanks!

**Comment by tiwillia** [comment]
@Miciah LGTM now!

File: documentation/oo_deployment_guide_comprehensive.adoc (hunk 1 above)
Comment (Miciah) [inline documentation/oo_deployment_guide_comprehensive.adoc]: Also, dropped two instances of "the", and changed "usrquota" to "quota_files" (thanks Tim!).

**Comment by tiwillia** [comment]
[merge]

**Pull request merged**